tests/VoiceProfileClientTests.ts

```diff
@@ -55,6 +55,7 @@ const BuildClient: (speechConfig?: sdk.SpeechConfig) => sdk.VoiceProfileClient =
 
 const BuildSpeechConfig: () => sdk.SpeechConfig = (): sdk.SpeechConfig => {
     const s: sdk.SpeechConfig = sdk.SpeechConfig.fromSubscription(Settings.SpeakerIDSubscriptionKey, Settings.SpeakerIDRegion);
+    // const s: sdk.SpeechConfig = sdk.SpeechConfig.fromHost(new URL(Settings.SpeakerIDDevHost), Settings.SpeakerIDSubscriptionKey);
```

Comment:
is this for the future? or like a debug toggle?

Reply:
debug toggle, i will remove once prod endpoint is deployed, the speaker ID key is updated, and tests pass again.